rebar.config.erl

```diff
@@ -41,7 +41,7 @@ quicer() ->
     {quicer, {git, "https://github.com/emqx/quic.git", {tag, "0.0.9"}}}.
 
 jq() ->
-    {jq, {git, "https://github.com/emqx/jq", {tag, "v0.2.2"}}}.
+    {jq, {git, "https://github.com/emqx/jq", {tag, "v0.3.1"}}}.
```

Comment:
I have pushed a new tag to fix a bug in ci which caused release package named without version number for certain platforms.
```suggestion
    {jq, {git, "https://github.com/emqx/jq", {tag, "v0.3.2"}}}.
```

Reply:
Got it, I'll update the version number.